react/index.web.js

```diff
@@ -9,12 +9,27 @@ import DialInSummaryApp from './features/invite/components/dial-in-summary/web/D
 import PrejoinApp from './features/prejoin/components/web/PrejoinApp';
 
 const logger = getLogger('index.web');
-const OS = Platform.OS;
+
+// Add global loggers.
+window.addEventListener('error', ev => {
+    logger.error(
+        `UnhandledError: ${ev.message}`,
+        `Script: ${ev.filename}`,
+        `Line: ${ev.lineno}`,
+        `Column: ${ev.colno}`,
+        'StackTrace: ', ev.error);
+});
+
+window.addEventListener('unhandledrejection', ev => {
+    logger.error(
```

Comment:
I've noticed that you've simplified the logic how do we get the stack and the error message for these handlers. I remember that sometime ago we had some troubles with the logger logging objects with some transports - rtcStats, and log transports based on post message. The reason was that we were trying to serialize the objects with JSON.stringify and that wasn't working very well for errors. 

Have you tested those if they work correctly with your implementation?

If they don't work correctly I guess we should either fallback to parsing the errors specifically here like in the old implementation. Or do some special handling in the logger transports or the logger.

Reply:
Unless they are circular reference including objects it should serialize just fine.

I'm not aware of any other problem that might arise during JSON serialization.

Reply:
The problem is that `JSON.stringify(new Error('test'))` will return `{}`.

Reply:
Oh I see. Lemme fix that.

Reply:
Fixed. Note that I'm building each string separately, so what will happen if the error objcet will get coerced to a string.

```
(new Error('aaaa')) + ''
"Error: aaaa" 
```

Reply:
Yes, not ideal but since we have the stack separately this wouldn't be an issue. 

I saw that before we were handling the case where the message argument has not been passed. We don't need this now?

Reply:
No because it's always there AFAICT.